scripts/two_line_bash.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+#
+#
+# This was casuing issues in tmux and always showed an "catkin_mil_ws", I'm sure that was intended
```

Comment:
That's the issue I caught last night, can't believe I missed it. If you want to check, you could pull the file in #10, but I'm sure it works now.